openid-connect-server-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -71,9 +71,9 @@
 			<trim-directive-whitespaces>true</trim-directive-whitespaces>
 		</jsp-property-group>
 	</jsp-config>
-	
+
 	<error-page>
-		<location>/error</location>
+		<location>/errorController</location>
```

Comment:
Can we still map this to `/error`?

Reply:
So I'm not sure this is possbile without breaking things. Some other jsp pages directly redirect/forwards to /error when changing the above "catch all" error location these wont get catch. Or do I misunderstand. Let me ask the otherway around:
- under what path should the ErrorController be reachable?
- under what path should the error.jsp be reachable?

another thing that should be done is to move the controller logic from error.jsp in the ErrorController.